oak_containers_stage1/src/main.rs

```diff
@@ -17,15 +17,25 @@
 #![feature(never_type)]
 
 mod image;
-mod server;
 
+use crate::proto::oak::containers::launcher_client::LauncherClient;
 use clap::Parser;
-use std::{error::Error, net::SocketAddr};
+use image::Image;
+use std::error::Error;
+use tonic::transport::{Channel, Uri};
+
+pub mod proto {
```

Comment:
nit, I'm not sure this should be public

Reply:
Yep, copy-paste failure, thanks for pointing it out.